scenarios.md

```diff
@@ -0,0 +1,199 @@
+
+## OCI Images & Artifacts
+
+The [OCI TOB][oci-tob] has adopted [OCI Artifacts][artifacts-repo], generalizing container images as one of many types of artifacts that may be stored in a registry. Other artifact types currently include:
+
+* [Helm Charts][helm-registry]
+* [Singularity][singularity]
+* Car firmware updates, deployed from OCI Artifact registries
+
+## The Need for a Generalized Signing Solution
+
+This document serves as the requirements and constraints of a generalized signing solution. It focuses on the scenarios and needs, and very specifically avoids any reference to other projects or implementations. As our working group forms a consensus on the requirements, the group will then transition to a spec.
+
+## Non-Goals
+
+- Notary v2 does not account for what the content represents or its lineage. Other efforts may attach additional content, and re-sign the super set of content to account for other scenarios. 
+
+## Key Stake Holders & Contributors
+
+As we identify the requirements and constraints, a number of key contributors will be asked to represent their requirements and constraints.
+
+> Please add companies, projects, products that you believe should be included.
+
+* Registry Cloud Operators
+  * [Azure Container Registry (acr)][acr] - Steve Lasker <steve.lasker@microsoft.com> (@stevelasker)
+  * [Amazon Elastic Container Registry (ecr)][ecr] - Omar Paul <omarpaul@amazon.com>
+  * [Docker Hub][docker-hub]
+  * [Google Container Registry (gcr)][gcr]
+  * [GitHub Package Registry (gpr)][gpr]
+  * [Quay][quay]
+  * [IBM Cloud Container Registry (icr)][icr]
+* Registry Vendors, Projects & Products
+  * [Docker Trusted Registry][docker-dtr]
+  * [Harbor][harbor]
+  * [JFrog Artifactory][jfrog]
+* Artifact Types
+  * Container Images - OCI Org
+  * Helm Charts
+  * Singularity
+  * Operator Bundles
+
+## Scenarios
+
+### Scenario #1: Local Build, Sign, Validate
+
+Prior to doing any deployment, a developer can test the: build, sign, validate scenario.
+
+1. Locally build a container image using a non-registry specific `name:tag`, such as:  
+  `$docker build hello-world:1`
+1. Locally sign `hello-world:1` 
+2. Run the image in a local environment which was configured to only support signed artifacts. *(**note** - details on the usability is deferred from this requirement. This could be a local docker host, a local kind or minikube instance configured to only run signed images)*  
+  `$ docker run hello-world:1`
+
+**Implications of this requirement:**
+
+- The local developer has access to signing keys
+- The local environment has a policy by which it states the set of keys it accepts.
+- The signing and validation of artifacts does not require a registry.
+- The signature used for validation may be hosted in a registry, or other accessible location.
+- The lack of a registry name does not infer docker.io as the default registry. This does imply changes to either the docker client, the notary client or both.
```

Comment:
> The lack of a registry name does not infer docker.io as the default registry. This does imply changes to either the docker client, the notary client or both.

I don't think this is the right takeaway.  We don't expect to change Docker's handling of bare names here, but rather that signing and verification can both happen locally without a registry.

Reply:
How do we tease out the question of an image being signed, without a registry name associated, but without the assumption it defaults to `docker.io`?

Reply:
Resolving this in lieu of the Definition & Terms @omieomye will be capturing.